match/lib/match/options.rb

```diff
@@ -292,6 +292,11 @@ def self.available_options
                                      description: "Renew the provisioning profiles if the certificate count on the developer portal has changed. Works only for the 'development' provisioning profile type. Requires 'include_all_certificates' option to be 'true'",
                                      type: Boolean,
                                      default_value: false),
+        FastlaneCore::ConfigItem.new(key: :renew_expired_certs,
+                                     env_name: "MATCH_RENEW_EXPIRED_CERTS",
+                                     description: "Automatically renew expired certificates. Notice: to renew developer_id and developer_id_installer certificates you must login with Account Holder account",
```

Comment:
Do we have now the ability to check if we have the right permission level, e.g. with spaceship? I don't remember... How does the code handle the failure if we don't have the right permissions level? Just a failure down the line when doing the operations on the dev portal? Any adverse side effect?

Reply:
Check: Using Tunes, we can match username with member email and get roles. But I never saw an endpoint to find a role assigned to the App Store Connect API Key.

Handling: just failure. Find out that the certificate is expired, delete in temp local decrypted folder, fail to generate one, and never reach the `save_changes!` line.

And I never saw fastlane checking user roles before doing some fetches or creating a certificate 😅.

Reply:
thanks for that explanation.